chef_master/source/resource_dnf_package.rst

```diff
@@ -0,0 +1,471 @@
+
+``provider``
+   **Ruby Type:** Chef Class
+
+   Optional. The chef-client will attempt to determine the correct provider during the chef-client run, and then choose the best/correct provider based on configuration data collected at the start of the chef-client run. In general, a provider does not need to be specified.
+
+``retries``
+   **Ruby Type:** Integer
+
+   The number of times to catch exceptions and retry the resource. Default value: ``0``.
+
+``retry_delay``
+   **Ruby Type:** Integer
+
+   The retry delay (in seconds). Default value: ``2``.
+
+``sensitive``
+  **Ruby Type** TrueClass, FalseClass
+
+   Ensure that sensitive resource data is not logged by the chef-client. Default value: ``false``. 
+
+``source``
+   **Ruby Type:** String
+
+   Optional. The path to a package in the local file system.
+
+``subscribes``
+   **Ruby Type:** Symbol, 'Chef::Resource[String]'
+
+   .. tag resources_common_notification_subscribes
+
+   A resource may listen to another resource, and then take action if the state of the resource being listened to changes. Specify a ``'resource[name]'``, the ``:action`` to be taken, and then the ``:timer`` for that action.
+
+   .. end_tag
+
+   .. tag resources_common_notification_timers
+
+   A timer specifies the point during the chef-client run at which a notification is run. The following timers are available:
+
+   ``:before``
+      Specifies that the action on a notified resource should be run before processing the resource block in which the notification is located.
+
+   ``:delayed``
+      Default. Specifies that a notification should be queued up, and then executed at the very end of the chef-client run.
+
+   ``:immediate``, ``:immediately``
+      Specifies that a notification should be run immediately, per resource notified.
+
+   .. end_tag
+
+   .. tag resources_common_notification_subscribes_syntax
+
+   The syntax for ``subscribes`` is:
+
+   .. code-block:: ruby
+
+      subscribes :action, 'resource[name]', :timer
+
+   .. end_tag
+
+``timeout``
+   **Ruby Types:** String, Integer
+
+   The amount of time (in seconds) to wait before timing out.
+
+``version``
+   **Ruby Types:** String, Array
+
+   The version of a package to be installed or upgraded. This attribute is ignored when using the ``:upgrade`` action.
+
+Multiple Packages
+-----------------------------------------------------
+.. tag resources_common_multiple_packages
+
+A resource may specify multiple packages and/or versions for platforms that use Yum, DNF, Apt, Zypper, or Chocolatey package managers. Specifing multiple packages and/or versions allows a single transaction to:
+
+* Download the specified packages and versions via a single HTTP transaction
+* Update or install multiple packages with a single resource during the chef-client run
+
+For example, installing multiple packages:
+
+.. code-block:: ruby
+
+   package ['package1', 'package2']
```

Comment:
We should update this to a cookstyle compliant setup:

package %w(package1 package2)